testutil/dbustest.go

```diff
@@ -58,6 +60,21 @@ type DBusTest struct {
 	SessionBus *dbus.Conn
 }
 
+const sessionBusConfigTemplate = `<busconfig>
```

Comment:
Maybe could use a comment:
```
// sessionBusConfigTemplate is a minimal session dbus daemon configuration template
// for use in the unit tests. In comparison to the typical disto session config, it does not
// specify  <standard_session_servicedirs/> to avoid accidental service activation.
```